### PR TITLE
Added chromedriver

### PR DESCRIPTION
**Pull request by mgualco-contractor**
tests: lib: common.py  
        added chrome_webproxy_driver method

### CODE REVIEW

**Comment by mattsm** [comment]
Can we not use chrome by default just yet?

**Comment by mgualco-contractor** [comment]
We could, we need to remove the "maximise" that we have in the current screen shot test (as in chrome is done by using the options)
I was thinking of adding this first, once merged we can start using it and make it the default. Unless unforseen issues crop up.

File: config.py [hunk 1]
```diff
@@ -35,3 +35,31 @@
 cdrouter_config = os.environ.get('BFT_CDROUTERCONFIG', None)
 cdrouter_wan_iface = os.environ.get('BFT_CDROUTERWANIFACE', "eth1")
 cdrouter_lan_iface = os.environ.get('BFT_CDROUTERLANIFACE', "eth2")
+
+
+# creates a small dictionary of all the options
+# this  will probably grow as options are added
+option_dict = {
+        "proxy":["nornal","sock5"],
```
Comment (mattsm) [inline config.py]: typo nornal

File: tests/screenshot_gui.py [hunk 2]
```diff
@@ -47,8 +47,11 @@ def runTest(self):
             raise Exception("No reasonable http proxy found, please add one to the board config")
 
         print("Using proxy %s" % proxy)
-        driver = lib.common.firefox_webproxy_driver(proxy)
-        driver.maximize_window()
+        #print("Using FirefoxDriver")
+        #driver = lib.common.firefox_webproxy_driver(proxy)
+        #driver.maximize_window()
+        print("Using ChromeDriver")
```
Comment (mattsm) [inline tests/screenshot_gui.py]: This change should be squashed with the newer change.

File: tests/screenshot_gui.py [hunk 3]
```diff
@@ -47,11 +47,8 @@ def runTest(self):
             raise Exception("No reasonable http proxy found, please add one to the board config")
 
         print("Using proxy %s" % proxy)
-        #print("Using FirefoxDriver")
-        #driver = lib.common.firefox_webproxy_driver(proxy)
-        #driver.maximize_window()
-        print("Using ChromeDriver")
-        driver = lib.common.chrome_webproxy_driver(proxy)
+        driver = lib.common.firefox_webproxy_driver(proxy)
```
Comment (mattsm) [inline tests/screenshot_gui.py]: I guess there should be one more commit selects chrome or ffox based on the new options?

**Comment by mgualco-contractor** [comment]
Now the screenshot can be done with either driver, via BFT_OPTIONS

**Comment by mattsm** [comment]
This was manually rebased and merged, please confirm things look OK and close this PR.

**Pull request closed without merging**